spec/classes/init_spec.rb

```diff
@@ -654,6 +654,68 @@
       }
     end
 
+    describe 'with ruby 2.2 with switch' do
```

Comment:
It seems Travis fails quite catastrophically on Puppet 4.x and Rubies 2.2 and 2.3. Not sure where to go from here.